patches/api/0428-Win-Screen-API.patch

```diff
@@ -18,10 +18,38 @@ index eb2fd6f0e09e50eeacfe4ceccf8fdede55c135a3..c68df5e5cc63b26da8623cf27e257ef0
 +     * and exits the end for the first time.
 +     * In vanilla, the win screen starts with a poem and then continues with the credits but its content can be
 +     * changed by using a resource pack.
++     * <br>
++     * Calling this method does not change the value of {@link #hasSeenWinScreen()}.
++     * That means that the win screen is still displayed to a player if they leave the end for the first time, even though
++     * they have seen it before because this method was called.
```

Comment:
Might be worth noting here that this method does not put players into the invulnerable 'won game' state players normally get while entering the portal and viewing the credits.